src/ServicePulse.Host/app/modules/monitoring/views/endpoint_details.html

```diff
@@ -111,8 +111,8 @@ <h1 class="righ-side-ellipsis col-lg-max-10" uib-tooltip="{{endpointName}}">
                     </div>
                     <div class="col-xs-6 no-side-padding scheduled-retries-rate-values">
                         <div class="row">
-                            <span class="metric-digest-header" uib-tooltip="Scheduled retry rate: The number of messages per second scheduled for retries (immediate or delayed).">
-                                Scheduled Retries Rate
+                            <span class="metric-digest-header" uib-tooltip="Retries rate: The number of messages per second scheduled for retries (immediate or delayed).">
```

Comment:
maybe an alternative would be "Scheduled retries:", that's also short but implies delayed retries (scheduled). There is a certain consistency using rate given the other columns though.